sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -891,106 +874,101 @@ describe("EventHub Receiver", function(): void {
+        "boo",
+        service.connectionString,
+        service.path
+      );
+      let subscription: Subscription | undefined;
+      const caughtErr = await new Promise<Error | MessagingError>((resolve) => {
+        subscription = badConsumerClient.subscribe(
+          {
+            processEvents: async () => {},
+            processError: async (err) => {
+              resolve(err);
+            }
+          },
+          { maxWaitTimeInSeconds: 2 }
+        );
       });
+      await subscription!.close();
+      await badConsumerClient.close();
 
-      it(`" " should throw an invalid EventHub address error`, async function(): Promise<void> {
-        try {
-          const id = " ";
-          debug("Created receiver and will be receiving messages from partition id ...", id);
-          const d = await client
-            .createConsumer(EventHubConsumerClient.defaultConsumerGroupName, id, latestEventPosition)
-            .receiveBatch(10, 3);
-          debug("received messages ", d.length);
-          throw new Error("Test failure");
-        } catch (err) {
-          debug("Receiver received an error", err);
-          should.exist(err);
-          err.message.should.match(
-            /.*Invalid EventHub address. It must be either of the following.*/gi
-          );
-        }
-      });
+      should.exist(caughtErr);
+      should.equal((caughtErr as MessagingError).code, "MessagingEntityNotFoundError");
+    });
 
-      const invalidIds2 = [""];
-      invalidIds2.forEach(function(id: string): void {
-        it(`"${id}" should throw an error`, async function(): Promise<void> {
-          try {
-            await client
-              .createConsumer(EventHubConsumerClient.defaultConsumerGroupName, id, latestEventPosition)
-              .receiveBatch(10, 3);
-            throw new Error("Test failure");
-          } catch (err) {
-            debug(`>>>> Received error - `, err);
-            should.exist(err);
+    it(`should throw an invalid EventHub address error for invalid partition`, async function(): Promise<
+      void
+    > {
+      let subscription: Subscription | undefined;
+      const caughtErr = await new Promise<Error | MessagingError>((resolve) => {
+        subscription = consumerClient.subscribe("boo", {
+          processEvents: async () => {},
+          processError: async (err) => {
+            resolve(err);
           }
         });
       });
+      await subscription!.close();
+      should.exist(caughtErr);
+      should.equal((caughtErr as MessagingError).code, "ArgumentOutOfRangeError");
     });
 
     it("should receive 'QuotaExceededError' when attempting to connect more than 5 receivers to a partition in a consumer group", function(done: Mocha.Done): void {
       const partitionId = partitionIds[0];
-      const rcvHndlrs: ReceiveHandler[] = [];
-      const rcvrs: any[] = [];
+      const subscriptions: Subscription[] = [];
```

Comment:
I'm kind of curious why we even have a test for this since this behavior is managed on the service-side and we don't do anything special (as far as I recall) with `QuotaExceededError` errors.

Reply:
Good point. And the service can change their quota any day and this test would then fail!

Reply:
Removed the test. Looking at history, it was a test case carried forward from Track 1 times.